Utilities/JIT.cpp

```diff
@@ -338,6 +338,7 @@ struct MemoryManager1 : llvm::RTDyldMemoryManager
 
 		if (!addr)
 		{
+			jit_log.fatal("JIT: Function '%s' linked but not found.", name);
```

Comment:
It's intentional to halt execution here, but allow continuing by pressing resume? If the isssue is fatal then allowing users to continue seems perplexing, and if this is a "continuable" problem then error would be more appropriate than fatal.

Or is the null simply caught later in process?

Reply:
Well I planned to create JIT trampoline that throws with actual function name only when it's called, not linked.

Reply:
Currently the (in my case) the error is logging during boot. So the game will start paused.
It doesn't really make sense to continue though, as it will crash later anyway in my case after the menu.